src/main/java/eu/europeana/metis/sandbox/controller/DatasetController.java

```diff
@@ -37,19 +37,23 @@
 
   private final ZipService zipService;
   private final DatasetService datasetService;
+  private final DatasetReportService reportService;
 
-  public DatasetController(ZipService zipService, DatasetService datasetService) {
+  public DatasetController(ZipService zipService,
+      DatasetService datasetService,
+      DatasetReportService reportService) {
     this.zipService = zipService;
     this.datasetService = datasetService;
+    this.reportService = reportService;
   }
 
   @ApiOperation("Process the given dataset")
   @PostMapping(value = "dataset/{name}/process", produces = APPLICATION_JSON_VALUE)
   @ResponseStatus(HttpStatus.ACCEPTED)
   public DatasetIdDto processDataset(
       @ApiParam(value = "name of the dataset", required = true) @PathVariable(value = "name") String datasetName,
-      @ApiParam(value = "country of the dataset", required = true, defaultValue = "NETHERLANDS") @RequestParam Country country,
-      @ApiParam(value = "language of the dataset", required = true, defaultValue = "NL") @RequestParam Language language,
+      @ApiParam(value = "country of the dataset", required = true, defaultValue = "Netherlands") @RequestParam Country country,
```

Comment:
Now the list of countries and languages will come from the variable value of the enum